.gitignore

```diff
@@ -2,3 +2,4 @@
 /output
 # python bytecode cache
 __pycache__
+.venv/
```

Comment:
I think this rather belongs into your personal gitignore. There is no established convention on how call the venv directory (I just call it plain `venv` and have that in my gitignore).